src/message/fetchActions.js

```diff
@@ -271,26 +270,46 @@ const fetchPrivateMessages = () => async (dispatch: Dispatch, getState: GetState
  * If the function is an API call and the response has HTTP status code 4xx
  * the error is considered unrecoverable and the exception is rethrown, to be
  * handled further up in the call stack.
+ *
+ * After a certain duration, times out with a TimeoutError.
```

Comment:
I think the following point will be moot if we move `MAX_TIME_MS` (with a rename) somewhere else, like in `config.js`, and justified its value a bit more. 🙂 But it's a chance to bring up one of our habits so far.

In the code you're looking at, I intentionally left `MAX_TIME_MS` out of the jsdoc. Our pattern with functions has been to use jsdoc for interface, and `//`-style comments for implementation. While I haven't always gotten this right, I think I might have in this case (though with the big asterisk in my previous paragraph 😉): someone reading the jsdoc can't resolve `MAX_TIME_MS` to a meaningful value without peeking under the hood and looking at its definition in the function's implementation. For that reason, I figured it didn't belong in the jsdoc.

I realized one thing quite a while after learning about that pattern: VSCode, at least with our config, parses a function's jsdoc and shows it to would-be callers on a hover interaction, etc. (I imagine other editors can maybe be made to do that too.) When I remember this, it can help me decide what's interface vs. implementation: a good jsdoc will help callers do their business without even having to open the file that the function is defined in.

I think the right way forward, here, might be to define the constant in `config.js`, in which case we can freely refer to it in the jsdoc because `config.js` is a scope that callers will naturally be aware of. Does that sound right? And @gnprice, how's my interpretation of the jsdoc / `//` pattern here; is it too rigid? 🙂